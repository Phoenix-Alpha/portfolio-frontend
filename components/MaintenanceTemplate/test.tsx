import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { MaintenanceTemplate } from '.'

describe('<MaintenanceTemplate />', () => {
  it('should renders correct <MaintenanceTemplate />', () => {
    const wrapper = shallow(<MaintenanceTemplate />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
