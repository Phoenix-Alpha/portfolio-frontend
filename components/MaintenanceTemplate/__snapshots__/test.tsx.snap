// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MaintenanceTemplate /> should renders correct <MaintenanceTemplate /> 1`] = `
<main
  className="flex flex-grow justify-center items-center min-h-screen bg-secondary"
>
  <div
    className="flex flex-col justify-center items-center py-12 md:py-[100px] px-12 md:px-[140px] mx-auto max-w-[80%]"
  >
    <p
      className="mx-auto font-body text-base md:text-2xl font-medium text-center text-white"
    >
      Under Maintenance
    </p>
  </div>
</main>
`;
